packages/grid/_modules_/grid/models/params/gridRowParams.ts

```diff
@@ -29,7 +29,21 @@ export interface GridRowParams<R extends GridRowModel = GridRowModel> {
 }
 
 /**
- * Object passed as parameter in the row getRowHeight callback.
+ * Object passed as parameter in the row `getRowClassName` callback prop.
+ */
+export interface GridRowClassNameParams extends GridRowParams {
+  /**
+   * Whether this row is the first visible or not.
+   */
+  isFirstVisible: boolean;
```

Comment:
Right now we mostly use the term "visible" to refer to the rows after sorting + filtering + children colapsing processed.

From what I understand, the `isFirstVisible` and `isLastVisible` is used here to talk about the paginated rows (and when pagination is off, all the sorted + filtered + collapsed rows).

I don't think we should use `visible` here.
Not in v5 because it conflicts with other usages of the term `visible` and makes it incoherent.
Not in v6 because I think we should stop using the term `visible` which is not clear (as we saw that each dev in the team has a different definition based on the day and the use case).

______

I would be in favor of using the term `page` even if the pagination is off since it's like it we had `pageSize=Infinity` and pagination on.

So `isFirstInPage` for instance.

But if you have a better idea I'm open to suggestion.
Just let's avoid term that can be understood in various ways.

Reply:
By using "visible" I wanted to adopt the same concept from [AG Grid](https://ag-grid.com/react-data-grid/grid-api/#reference-displayedRows). The visible (or displayed) rows are those you can see by only scrolling the grid. In the past it was heavily misused but I would ignore and start using "visible" in this way from now.

Reply:
If we are acting that `visible` should mean "accessible by scroll"
Then could we 

1. Create an issue for renaming everything to this new nomenclature or at least put a line in #3287 
2. Be more precise in the description of this prop, at least until v6 `Wheter this row is the first row reachable by scrolling or not.` or something similar. So that we are sure people understand `visible` in the correct way